openslides_backend/action/actions/committee/update.py

```diff
@@ -20,5 +26,58 @@ class CommitteeUpdateAction(UpdateAction):
             "user_ids",
             "forward_to_committee_ids",
             "receive_forwardings_from_committee_ids",
+            "organisation_tag_ids",
         ]
     )
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if has_organisation_management_level(
+            self.datastore, self.user_id, OrganisationManagementLevel.SUPERADMIN
+        ):
+            return
+
+        cml_field = f"committee_${instance['id']}_management_level"
+        user = self.datastore.get(
+            FullQualifiedId(Collection("user"), self.user_id), [cml_field]
+        )
+        is_manager = user.get(cml_field) == "can_manage"
```

Comment:
Please use the enum here

Reply:
Also, you should use `>=` in case we add more levels. For that you have to create a CommitteeManagementLevel first:
```
CommitteeManagementLevel(user.get(cml_field, "no_right")) <= CommitteeManagementLevel.CAN_MANAGE
```

Reply:
Done.

Reply:
CommitteeManagementLevel.CAN_MANAGE does not exist and the constructor doesn't work that way.

Reply:
> CommitteeManagementLevel.CAN_MANAGE does not exist

Ah, it is currently named `MANAGER` -  this probably slipped through the review. Please rename to `CAN_MANAGE` for consistency.



> and the constructor doesn't work that way.

Why not? For OML, it does work: https://github.com/OpenSlides/openslides-backend/blob/a32ee0a36cfcc9d4cb6695aa66595b2a74c2be61/openslides_backend/action/actions/user/create_update_permissions_mixin.py#L63, so I thought CML should work analogously.

Reply:
It throws an ugly type error.

Reply:
This should work, too. I think I will look into how to unify the management levels another time

Reply:
CommitteeManagementLevel(user.get(cml_field, "no_right"))
shows a:
openslides_backend/action/actions/committee/update.py:47: error: Too few arguments for "CommitteeManagementLevel"
in flake8.

Reply:
This can just be ignored, that's why there's a `type: ignore` at all places for the OML, too